tests/doc_test/filter_doc/filter_no_needs.rst

```diff
@@ -0,0 +1,26 @@
+filter_no_needs
+===============
+
+.. req:: filter_warning_req_a
+   :tags: 1;
+   :status: open
+   :hide:
+
+.. req:: filter_warning_req_b
+   :tags: 2;
+   :status: closed
+
+.. needtable::
+   :filter: ("5" in tags)
+
+.. needtable::
+   :filter: ("6" in tags)
+   :filter_warning: No required needs found in table
+
+.. needlist::
+   :tags: 4711
+   :filter_warning: No required needs found in list
+
+.. needtable::
+   :filter: ("7" in tags)
+   :filter_warning: 
```

Comment:
Would add here a positive filter result as well, to be sure the text is not printed if needs got found.